Fill.py

```diff
@@ -25,7 +25,7 @@ def sweep_from_pool(base_state: CollectionState, itempool):
 def fill_restrictive(world: MultiWorld, base_state: CollectionState, locations, itempool: typing.List[Item],
                      single_player_placement=False, lock=False):
     unplaced_items = []
-    placements = []
+    placements: list[Location] = []
```

Comment:
in python 3.8 this needs to be typing.List

Reply:
Fixed